frontend/src/shared/consts/playerPositionsOrder.ts

```diff
@@ -0,0 +1,10 @@
+import type { PlayerPositions } from "../../entity";
+
+type PlayerPositionsOrder = { [n in PlayerPositions]: number };
+
+export const playerPositionsOrder: PlayerPositionsOrder = {
```

Comment:
давай попробуем тут использовать enum
```suggestion
export enum PlayerPosition {
  Goalkeeper,
  Defender,
  Midfielder,
  Attacker,
}
```